src/main/resources/application.yml

```diff
@@ -105,8 +96,29 @@ observe:
     hearingLabel:
       path: "hearing.prosecutionCases[*].defendants[*].offences[*].judicialResults[*].label"
       printValue: true
-    hearingResultText:
```

Comment:
Removed  the contents can be quite wordy and has things like email addresses in it which we shouldn't be logging.